alacritty/src/window.rs

```diff
@@ -135,13 +146,20 @@ fn create_gl_window(
     Ok(windowed_context)
 }
 
-/// A window which can be used for displaying the terminal
+/// A window which can be used for displaying the terminal.
 ///
-/// Wraps the underlying windowing library to provide a stable API in Alacritty
+/// Wraps the underlying windowing library to provide a stable API in Alacritty.
 pub struct Window {
     windowed_context: WindowedContext<PossiblyCurrent>,
     current_mouse_cursor: CursorIcon,
     mouse_visible: bool,
+    /// Wayland setting to control redrawing of the window, according to the frame callbacks.
```

Comment:
```suggestion
    /// Flag tracking frame redraw requests from Wayland.
```

Maybe this is a bit better? My main goal is just to shorten this comment down a tiny bit, it seems very long for just a single flag which might make it harder to understand. Though I'm not certain we should change it.